Gems/WhiteBox/Code/Source/WhiteBoxToolApiReflection.cpp

```diff
@@ -81,107 +81,200 @@ namespace WhiteBox
             behaviorContext->Class<WhiteBoxMeshHandle>("WhiteBoxMeshHandle")
                 ->Attribute(AZ::Script::Attributes::Scope, AZ::Script::Attributes::ScopeFlags::Automation)
                 ->Attribute(AZ::Script::Attributes::Module, "whitebox.api")
+                ->Method("IsValid", [](WhiteBoxMeshHandle* whiteBoxMeshHandle) {
+                    return (WhiteBoxMeshFromHandle(*whiteBoxMeshHandle) != nullptr);
+                })
```

Comment:
nit/standards: Braces on new line (also can omit the `() around the expression, overkill here)

Reply:
Also we should probably default initialize `m_whiteBoxMeshAddress` to `0` as a member initializer in `WhiteBoxMeshHandle` to ensure this works as expected